_pages/team.html

```diff
@@ -9,52 +9,52 @@
     <h1>Our Members</h1>
     <div>
         <div>
-            <img src="/assets/img/team/ben-fong.png">
+            <img src="/assets/img/team/ben-fong.png" alt="A pixelated portrait of Ben Fong in a red t-shirt.">
```

Comment:
I probably should have mentioned this: We don't need alt tags on team photos or robot photos, because they have a descriptive title immediately underneath them.

Reply:
So the robot pages don't need alt tags at all?